optional.tf

```
@@ -277,16 +289,16 @@ variable "vpc_id" {
   type        = string
 }
 
-variable "primary_hosted_zone" {
-  description = "Name of the primary hosted zone for DNS. e.g. primary_hosted_zone = example.org --> service.example.org. If null, it is assumed that a private hosted zone will be used."
+variable "hosted_zone" {
+  description = "Name of the hosted zone for DNS. e.g. hosted_zone = example.org --> service.example.org. Based on the is_private, this is the primary or the private hosted zone."
   default     = null
   type        = string
 }
 
-variable "private_hosted_zone" {
-  description = "Name of the private hosted zone for DNS. e.g. private_hosted_zone = example.org --> service.example.private. If null, it is assumed that a public hosted zone will be used."
-  default     = null
-  type        = string
+variable "is_private" {
```

```suggestion
variable "is_hosted_zone_private" {
```

This makes more sense to me, as it confused me why we were setting both `is_private` and `public_service` here:
https://github.com/pbs/terraform-aws-ecs-service-module/pull/93/files#diff-db0fce55d86d79a0ec71af0d6d55a52b35c4eb98271d6431fbc6054d756d8788